tests/ert/unit_tests/resources/test_run_eclipse_simulator.py

```diff
@@ -15,148 +11,103 @@
-            "SLBSLS_LICENSE_FILE": "7321@eclipse-lic-no.statoil.no",
-            "ECLPATH": "/prog/res/ecl/grid",
-            "PATH": "/prog/res/ecl/grid/macros",
-            "F_UFMTENDIAN": "big",
-            "LSB_JOBID": None,
-        }
-    }
-
-
-@pytest.fixture
-def init_eclrun_config(tmp_path, monkeypatch, eclrun_conf):
-    with open(tmp_path / "ecl100_config.yml", "w", encoding="utf-8") as f:
-        f.write(yaml.dump(eclrun_conf))
-    monkeypatch.setenv("ECL100_SITE_CONFIG", "ecl100_config.yml")
-
-
-def test_get_version_raise():
-    econfig = ecl_config.Ecl100Config()
-    class_file = inspect.getfile(ecl_config.Ecl100Config)
-    class_dir = os.path.dirname(os.path.abspath(class_file))
-    msg = os.path.join(class_dir, "ecl100_config.yml")
-    with pytest.raises(ValueError, match=msg):
-        econfig._get_version(None)
-
-
-@pytest.mark.usefixtures("use_tmpdir", "init_eclrun_config")
-def test_eclrun_will_prepend_path_and_get_env_vars_from_ecl100config(
-    eclrun_conf,
-):
-    # GIVEN a mocked eclrun that only dumps it env variables
-    Path("eclrun").write_text(
-        textwrap.dedent(
-            """\
-                #!/usr/bin/env python
-                import os
-                import json
-                with open("env.json", "w") as f:
-                    json.dump(dict(os.environ), f)
-                """
-        ),
-        encoding="utf-8",
-    )
-    os.chmod("eclrun", os.stat("eclrun").st_mode | stat.S_IEXEC)
-    Path("DUMMY.DATA").write_text("", encoding="utf-8")
-    econfig = ecl_config.Ecl100Config()
-    eclrun_config = ecl_config.EclrunConfig(econfig, "dummyversion")
-    erun = ecl_run.EclRun("DUMMY", None, check_status=False)
-    with mock.patch.object(
-        erun, "_get_run_command", mock.MagicMock(return_value="./eclrun")
+@pytest.fixture(name="e100_env")
+def e100_env(monkeypatch):
+    for var, value in (
+        ErtPluginManager()
+        .get_forward_model_configuration()
+        .get("ECLIPSE100", {})
+        .items()
     ):
-        # WHEN eclrun is run
-        erun.runEclipse(eclrun_config=eclrun_config)
-
-    # THEN the env provided to eclrun is the same
-    # as the env from ecl_config, but PATH has been
-    # prepended with the value from ecl_config
-    with open("env.json", encoding="utf-8") as f:
-        run_env = json.load(f)
-
-    expected_eclrun_env = eclrun_conf["eclrun_env"]
-    for key, value in expected_eclrun_env.items():
-        if value is None:
-            assert key not in run_env
-            continue  # Typically LSB_JOBID
-
-        if key == "PATH":
-            assert run_env[key].startswith(value)
-        else:
-            assert value == run_env[key]
+        monkeypatch.setenv(var, value)
+    yield
 
 
 @pytest.mark.integration_test
-@pytest.mark.usefixtures("use_tmpdir", "init_eclrun_config")
+@pytest.mark.usefixtures("use_tmpdir", "e100_env")
 @pytest.mark.requires_eclipse
 def test_ecl100_binary_can_produce_output(source_root):
```

Comment:
Generally the names here say something about what ecl100 can do, we should probably rephrase all of these to be about what we expect from `run_reservoirsimulator`.